provisioning/service/src/Config/IndividualEnrollment.cs

```diff
@@ -212,6 +212,12 @@ public override string ToString()
         [JsonProperty(PropertyName = "registrationState", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public DeviceRegistrationState RegistrationState { get; private set; }
 
+        /// <summary>
+        /// Optional Device Information.
```

Comment:
What kind of information would a user put or read from here? How will it be processed? How should it be interpreted?

Reply:
It's basically just a Map, it's for arbitrary information a customer might want to store so there's no real specific use case. It was added to reconcile the SDK with the current API version since I updated it. I can move it into a different PR if that would be better since it's not strictly related to RBAC

Reply:
moved to https://github.com/Azure/azure-iot-sdk-csharp/pull/2299

Reply:
The reason I ask is for our customers. The doc comments should give them more info. How should I (as an SDK user) use this? With this SDK, do I set these key value pairs? Are they interpreted by the DPS service? Will the device have access to them? It would be good to have helpful doc comments to guide the customer.